### DISCUSSION TITLE
Zapier Support

### DISCUSSION THREAD

**Discussion by cmcaboy** [planetscale/discussion#519] [General]
Is Zapier supported with planetscaleDB? If so, is there any details for us to get started with an integration?

**Comment by orware** [comment]
Hello @cmcaboy,

We do have an older discussion question over in [discussions#179](https://github.com/planetscale/discussion/discussions/179) which made it sound like it wasn't possible to connect last year, but things may be a bit better now so I think it's worth seeing if you'll be more successful.

Based on the details shown here:
https://help.zapier.com/hc/en-us/articles/8495937131533-How-to-Get-Started-with-MySQL#connect-to-app-on-zapier

It seems like it may be possible as the SSL/TLS configuration settings seem to be available there.

You would use the public credentials created within the PlanetScale UI primarily, and fill in the hostname, database name, username, and password fields for your PlanetScale database in Zapier. 

After that, you would want to make sure "Enforce encryption" is enabled, and then you could leave the "SSL client certificate" blank as well as the "SSL private key", but you would want to fill in the "SSL CA bundle" field.

For the "SSL CA Bundle" field you could grab the Let's Encrypt Root CA certificate from the following URL first:
https://letsencrypt.org/certs/isrgrootx1.pem

And open up that file in a text editor so you can see the contents since you will need to copy/paste the text in to Zapier as I don't know if their UI will allow you to upload the file directly.

Definitely give that a try if you can and let me know how it goes!
